src/commands/wrap.ts

```diff
@@ -46,16 +43,45 @@ export const builder = (yargs: Argv): Argv =>
       description: "Use if raw directory contains wrapped files",
       type: "boolean",
       default: false
+    })
+    .option("output-file", {
+      alias: "of",
+      type: "string",
+      description: "Use if output path is a file",
+      conflicts: "output-dir"
+    })
+    .option("output-dir", {
+      alias: "od",
+      type: "string",
+      description: "Use if output path is a directory",
+      conflicts: "output-file"
     });
 
 export const handler = async (args: WrapCommand): Promise<string> => {
   try {
-    const merkleRoot = await wrap(args.rawDocumentsPath, args.wrappedDocumentsDir, {
+    const outputPathType = args.outputDir ? "directory" : args.outputFile ? "file" : "stdOut";
+    const outputPath =
+      outputPathType === "directory" ? args.outputDir : outputPathType === "file" ? args.outputFile : "stdOut";
+
+    // when input type is directory, output type must only be directory
+    if (isDir(args.rawDocumentsPath) && outputPathType !== "directory") {
+      signale.error(
+        "Output path type can only be directory when using directory as raw documents path, use --output-dir"
+      );
+      process.exit(1);
+    }
+
+    const merkleRoot = await wrap(args.rawDocumentsPath, outputPath, {
       schemaPath: args.schema,
       version: args.openAttestationV3 ? "open-attestation/3.0" : "open-attestation/2.0",
-      unwrap: args.unwrap
+      unwrap: args.unwrap,
+      outputPathType
     });
-    signale.success(`Batch Document Root: 0x${merkleRoot}`);
+
+    if (outputPathType !== "stdOut") {
```

Comment:
I feel this is not good enough. we need a global mechanism to disable the logging based on a condition

Reply:
Note that we still need to display errors :)

Reply:
i don't understand this one